homework7/7.1/CalculatorTest/CalculatorTest.cs

```diff
@@ -0,0 +1,69 @@
+using NUnit.Framework;
+using System;
+using Task7_1;
+
+namespace CalculatorTest;
+
+public class Tests
+{
+    private Calculator calculator = new();
+
+   [SetUp]
+    public void SetUp()
+    {
+        calculator = new Calculator();
```

Comment:
```suggestion
        calculator = new();
```
Не уверен, что так _нужно_ (тяжело читать код, где создаётся _что-то_ непонятное, а не Calculator), но так определённо можно.